src/app/(auth)/(navigationsBarLayout)/mypage/ui/ModifyForm.tsx

```
@@ -0,0 +1,164 @@
+              placeholder="닉네임을 입력하세요"
+            />
+          </div>
+
+          {/* GitHub 주소 */}
+          <div>
+            <label className="block text-sm font-medium text-gray-300 mb-2">GitHub 주소</label>
+            <input
+              type="url"
+              value={editForm.githubUrl || ''}
+              onChange={(e) =>
+                setEditForm((prev) => ({
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)